src/components/common/searchbar/index.js

```diff
@@ -53,7 +53,7 @@ function SearchBar({ searchVisible, setData }) {
           placeholder="Search through your inventory"
           className="searchBar"
           onSearch={onSearch}
-          onChange={onChange}
+          onChange={e => onSearch(e.target.value)}
```

Comment:
We have already onChange function:
 function onChange(e) {
    setData(e.target.value);
  }
So, we do not need this change.

Reply:
Alright. Just wanting to try and filter it down as the user is typing, But it is probably better after they enter the full term.